di_website/publications/models.py

```diff
@@ -1,57 +1,56 @@
-)
-from wagtail.admin.edit_handlers import FieldPanel, InlinePanel, StreamFieldPanel, PageChooserPanel
-from wagtail.snippets.models import register_snippet
-from wagtail.snippets.edit_handlers import SnippetChooserPanel
+from num2words import num2words
+from taggit.models import Tag, TaggedItemBase
+from wagtail.admin.edit_handlers import (FieldPanel, InlinePanel,
+                                         PageChooserPanel, StreamFieldPanel)
 from wagtail.contrib.redirects.models import Redirect
 from wagtail.contrib.search_promotions.templatetags.wagtailsearchpromotions_tags import \
     get_search_promotions
-from wagtail.images.edit_handlers import ImageChooserPanel
-from wagtail.images.blocks import ImageChooserBlock
+from wagtail.core import hooks
+from wagtail.core.blocks import (CharBlock, PageChooserBlock, StructBlock,
+                                 URLBlock)
+from wagtail.core.fields import RichTextField, StreamField
+from wagtail.core.models import Orderable, Page
 from wagtail.documents.edit_handlers import DocumentChooserPanel
-from wagtail.search.models import Query
 from wagtail.embeds.blocks import EmbedBlock
+from wagtail.images.blocks import ImageChooserBlock
+from wagtail.images.edit_handlers import ImageChooserPanel
+from wagtail.search.models import Query
+from wagtail.snippets.edit_handlers import SnippetChooserPanel
+from wagtail.snippets.models import register_snippet
 from wagtailmedia.edit_handlers import MediaChooserPanel
 
-from di_website.common.base import hero_panels, get_paginator_range, get_related_pages
-from di_website.common.mixins import HeroMixin, OtherPageMixin, SectionBodyMixin, TypesetBodyMixin
-from di_website.common.constants import MAX_PAGE_SIZE, MAX_RELATED_LINKS, RICHTEXT_FEATURES
+from di_website.common.base import (get_paginator_range, get_related_pages,
+                                    hero_panels)
+from di_website.common.constants import (MAX_PAGE_SIZE, MAX_RELATED_LINKS,
+                                         RICHTEXT_FEATURES)
+from di_website.common.mixins import (HeroMixin, OtherPageMixin,
+                                      SectionBodyMixin, TypesetBodyMixin)
 from di_website.downloads.utils import DownloadsPanel
 
-from taggit.models import Tag, TaggedItemBase
-from .mixins import (
-    FlexibleContentMixin, UniqueForParentPageMixin, PageSearchMixin, LegacyPageSearchMixin, ParentPageSearchMixin,
-    PublishedDateMixin, UUIDMixin, ReportChildMixin, FilteredDatasetMixin)
-from .utils import (
-    ContentPanel, PublishedDatePanel, WagtailImageField,
-    UUIDPanel, get_first_child_of_type, get_ordered_children_of_type, get_downloads)
 from .edit_handlers import MultiFieldPanel
 from .inlines import *
-
+from .mixins import (FilteredDatasetMixin, FlexibleContentMixin,
+                     LegacyPageSearchMixin, PageSearchMixin,
+                     ParentPageSearchMixin, PublishedDateMixin,
+                     ReportChildMixin, UniqueForParentPageMixin, UUIDMixin)
```

Comment:
You should get the `guides` and `indent-rainbow` extensions to help you highlight messed up indentations ... this looked better in its original format... as did some others.

Reply:
Updated indentation for imports